### DISCUSSION TITLE
Watched dir (smb/cifs mount) not working

### DISCUSSION THREAD

**Discussion by GV-5g** [transmission/transmission#8383] [Support]
### What is the issue?

Ubuntu 22.04.2 LTS (jammy)

Switched from rakshasa's rTorrent 0.9.8 to Transmission 4.0.0 (GTK).

.torrent files are not auto-loaded from the watched directory until moved/renamed. The directory is a `cifs` mount (an NTFS folder shared by the Windows VM host.)

It appears that `inotify` doesn't work on `cifs` mounts when files are added remotely. This was not an issue for rTorrent, which promptly detected new .torrent files regardless.

### Which application of Transmission?

GTK+ app on Linux, BSD, etc.

### Which version of Transmission?

4.0.0

**Comment by jiggol** [comment]
After stopping the daemon, set the value of "watch-dir-force-generic": true in the settings.json file, and then restart the daemon.

https://forum.transmissionbt.com/viewtopic.php?p=74157#p74157

**Comment by mikedld** [comment]
The `watch-dir-force-generic` is only applicable to the daemon. GTK client has its own implementation based on [`Gio::FileMonitor`](https://docs.gtk.org/gio/class.FileMonitor.html).Same is true for Qt and Mac clients which have their own implementations too. At one point I thought of using the common implementation from libtransmission (which is used by the daemon) but it requires some work and decisions to be made.
